tools/crm_resource.c

```diff
@@ -825,11 +825,7 @@ main(int argc, char **argv)
         resource_t *rsc = pe_find_resource(data_set.resources, rsc_id);
         node_t *dest = pe_find_node(data_set.nodes, host_uname);
 
-        rc = -ENXIO;
```

Comment:
We need to keep this line for when rsc or dest is NULL.

Unrelated to your changes, it looks like the rsc == NULL case here should also goto bail, but doesn't due to an earlier copy-and-paste error.

Reply:
right 👍 ..i thought that it would be overwritten by cli_resource_ban at 835.
But now that i had relook i see that if either rsc or dest is NULL it would go over to bail.
Nice Catch.
I will the rc = -ENXIO; and also the fix the bug of the missing bail.
I also seem to have broken a test case with the recent changes.
Is it ok if i squash the commits after fixing all the above mentioned changes ? 😄

Reply:
Of course, no problem. I think the broken test will be fixed by the -ENXIO change.